doc/source/_static/custom.css

```diff
@@ -61,3 +61,39 @@ img.logo {
 .prev-next-top {
   margin-bottom: 1em;
 }
+
+/* Sidebar TOC and headers */
+
+div.sphinxsidebarwrapper div {
+    margin-bottom: .8em;
+}
+div.sphinxsidebar h3 {
+    font-size: 1.3em;
+    padding-top: 0px;
+    font-weight: 800;
+    margin-left: 0px !important;
+}
+
+div.sphinxsidebar p.caption {
+    font-size: 1.2em;
+    margin-bottom: 0px;
+    margin-left: 0px !important;
+    font-weight: 800;
+    color: gray;

```

Comment:
I vaguely remember @carreau had feelings about `!important` but I'm not sure why.

Reply:
the reason that we're using it here is because `!important` is used above in the CSS so it's overriding everything else...which is probably both the reason why using `!important` is discourage, and also the reason why we need to use it here :-P

Reply:
Yeah... if really we get too many `!important` we can decide to refactor.